core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/aggregate/FinalAggregateOpExec.scala

```diff
@@ -31,33 +34,48 @@ class FinalAggregateOpExec[Partial <: AnyRef](
       pauseManager: PauseManager,
       asyncRPCClient: AsyncRPCClient
   ): Iterator[Tuple] = {
+    if (aggFuncs.isEmpty) {
+      throw new UnsupportedOperationException("Aggregation Functions Cannot be Empty")
```

Comment:
Please add a check to throw this exception also in `SpecializedAverageOpDesc`, inside the `operatorExecutor` function, so this error can be caught earlier.